app/views/simulator/mobile_disabled.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-lg-6 col-md-6 middle-block">
+      <%= image_tag "SVGs/mobile-user.svg",alt: "mobile-user", :class => "signup-cover center-block" %>
+      <div class="welcome-text">
+        <label>Welecome Back, Mobile User</label>
+      </div>
+    </div>
+
+    <div class="col-lg-6 col-md-6 middle-block">
+      <div class="form-container">
+        <h1 class="pb-3">Simulator Disabled</h1>
+        <p>
+        Unfortunately, CirciutVerse simulator is disabled on mobile devices.
+        You can still use it on your computer.
+        We decided to disable it cause it was hard to use on small screens.
+        Sorry for the inconvenience
+        </p>
+        <button id='go-back-button' type="button" class="btn btn-primary btn-lg btn-block">Go back</button>
+        <div class='text-center'><a id='continue-anyway' lass='font-weight-light' style='color:inherit;'>Continue anyway</a><div>
+      </div>
+    </div>
+  </div>
+</div>
+<script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
```

Comment:
I don't think we need a `continue anyway` function or cookie - just disable it altogether

Reply:
In case of bad device detection or emergency fixes we should allow users to pass